topics/transcriptomics/tutorials/differential-isoform-expression/slides.html

```diff
@@ -0,0 +1,257 @@
+    - Peptides smaller than 100 aminoacids are usually discarted
+- Frequently annotated as non-coding RNAs
+- Propensity for structural disorder
+    -  Discarted as intrinsically disordered proteins (IDPs)
+
+---
+
+## Why study small peptides?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/small_peptides.png %})
+
+---
+
+## Why study small peptides?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/short_peptides.png %})
+
+---
+
+## Why study small peptides?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/small_peptides_examples.png %})
+
+.footnote[Source: Steinberg, R., & Koch, H. G. (2021). The largely unexplored biology of small proteins in pro- and eukaryotes. The FEBS journal, 288(24), 7002-7024]
+
+---
+
+## Annotated as non-coding RNAs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/question.jpg %})
+
+---
+
+## Annotated as non-coding RNAs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/small_non_coding.png %})
+
+---
+
+## Annotated as non-coding RNAs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/spencer_datase.png %})
+
+---
+
+## Why study intrinsically disordered proteins (IDP)?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/disordered_regions.png %})
+
+.footnote[Source: Babu, M. M., Kriwacki, R. W., & Pappu, R. V. (2012). Versatility from protein disorder. Science, 337(6101), 1460-1461.]
+
+---
+
+## Disorder-Function Paradigm
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/disorder_article.png %})
+
+---
+
+## Disorder-Function Paradigm
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/intrinsically_disordered.png %})
+
+---
+
+## Disorder-Function Paradigm
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/disorder_protein.jpg %})
+
+.footnote[Source: Chakrabarti, P., & Chakravarty, D. (2022). Intrinsically disordered proteins/regions and insight into their biomolecular interactions. Biophysical Chemistry, 283, 106769]
+
+---
+
+## How identify non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/scientist.jpg %})
+
+---
+
+## How identify non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/isoform_switching.png %})
+
+---
+
+## How identify non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/isoform_switching.jpg %})
+
+---
+
+## Galaxy Workflow
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/galaxy_training.png %})
+
+Full detailed explanation in the [Genome-wide alternative splicing analysis](https://gxy.io/GTN:T00345) Galaxy training.
```

Comment:
if you're linking to a GTN material internally we would prefer using {% link topics/... %} if possible

Reply:
Sure, I forgot about that.